cmd/serve.go

```diff
@@ -12,15 +14,29 @@ var serveCmd = &cobra.Command{
 	Use:   "serve",
 	Short: "Starts the HTTP REST APIs server",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		webBuilder := new(config.WebBuilder).InitFromViper(viper.GetViper())
-		server := new(web.Server).InitFromWebBuilder(webBuilder)
+		mailChannel := make(chan mail.Mail)
+
+		builder := new(config.WebBuilder).InitFromViper(viper.GetViper())
+
+		resources.Outbox = mailChannel
+		resources.BaseUIPath = builder.BaseUIPath
+
+		mailHandler := new(mail.DefaultApi).Init(builder.MailUser, builder.MailPassword, builder.MailHost, builder.MailPort, builder.MailIdentity, mailChannel)
```

Comment:
@claudiosegala, please rename `DefaultApi` to `DefaultHandler`. Also, there are too many parameters. Shouldn't you just pass a WebBuilder and the mailChannel?

Reply:
@abilioesteves I tried to use web builder, but it created a cycle dependency. I can try again.